### PR TITLE
ceremony to bridge assets with karura and some docs

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**
- steps to start transfers with karura parachain
- some docs on how to test chains
- would you be able to run using this doc?

### CODE REVIEW

**Review by hussein-aitlahcen — DISMISSED** [review]

File: scripts/polkadot-launch/awesome-polkadot-hosting.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Overview
+
+To run, any chain needs:
+
+- genesis state (to be shared with cluster of nodes which form consensus)
+- runtime to execute (logic of)
+- bootsrap networking information (which nodes to connect)
```
Comment (Dom-Roth) [inline scripts/polkadot-launch/awesome-polkadot-hosting.md:7]:
```suggestion
- bootstrap networking information (which nodes to connect)
```
Typo

File: scripts/polkadot-launch/awesome-polkadot-hosting.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+- and, either relay setup or what relay to use
+- and other utilitary setup like loggin, metrics
```
Comment (Dom-Roth) [inline scripts/polkadot-launch/awesome-polkadot-hosting.md:9]:
```suggestion
- and other utilitary setup like logging, metrics
```
Typo

File: scripts/polkadot-launch/awesome-polkadot-hosting.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+- private keys in keystore 
+- logical netwrok connectivty allowance (bridges)
```
Comment (Dom-Roth) [inline scripts/polkadot-launch/awesome-polkadot-hosting.md:11]:
```suggestion
- logical network connectivity allowance (bridges)
```
Typo

**Review by Dom-Roth — APPROVED** [review]
LGTM!
Clear & understandable to me, I could start working with this.

**Review by Dom-Roth — APPROVED** [review]

**Review by hussein-aitlahcen — APPROVED** [review]

**Comment by hussein-aitlahcen** [comment]
bors +r

**Comment by dzmitry-lahoda** [comment]
r+

**Pull request merged**